### ISSUE TITLE
Weekly Status 2017-04-10

### ISSUE THREAD

**Issue by mrjoro** [ampproject/amphtml#8654]
Please add a comment to this issue with your weekly status update for the AMP Project, such as:

- your accomplishments over the last week
- what you're currently working on
- things you're blocked on
- anything else you think the entire community should know

We encourage updates from everyone who is actively participating in the AMP Project.

**Comment by DiegoAche** [comment]
Hello,
I don't know if this is the place to put this, or if is better to open a new issue. Since las saturday, i think, all the files form the cdn didn't load in our website. Last friday everything worked perfect but for some reason all the js files of amp dont load. I dont known if you change the path to the files and didn't update on the documentation or is some problem with our web:

https://www.juguetronica.com/amp/

Could you help me please

Thanks!!

**Comment by DiegoAche** [comment]
It has already been fixed, looks like it was a problem with google cdn. 

Thanks anyway ;)

**Comment by mrjoro** [comment]
Glad your issues was resolved @DiegoAche; in the future if you have an issue there's some info on where to get help here:  https://www.ampproject.org/support/

(This issue is intended as a place for people working on AMP Project contributions to post updates.)

**Comment by cramforce** [comment]
- Fixed nasty race that broke our pre-render perf numbers #8630
- Limited number of elements we ever build during prerender #8364
- Made tests less flaky by removing font references.

**Comment by adelinamart** [comment]
- Looking into our Q1 Github issue metrics: ~ 800 issues closed

**Comment by choumx** [comment]
- Discussed approaches for filtering/sorting (#8368)
- Support more flexible e-commerce use cases in `amp-bind` (#8648)
- Added debugging and error types to `amp-bind` docs (#8549)

**Comment by aghassemi** [comment]
- `amp-sort`, `amp-filter`, `height-responsive` layout brainstorming, design and prototyping.
- experiment flags cleanups

**Comment by kmh287** [comment]
- Added event data for change events on slider, radio, and checkbox inputs #8569 
- Added custom array function to bind to make modifying arrays cleaner #8485
- Changed bind to use a deep merge algorithm instead of shallow merge for state updates #8458 
- Working on additions to bind documentation #8660 
- Working on a race condition between amp-form and bind #8661

**Comment by cvializ** [comment]
- amp-form without mustache templates #8565 
- deflaked amp-form integration tests #8577
- design doc for asynchronous form validation

**Comment by zhouyx** [comment]
- Fixed issue on passing user defined port number to gulp server. #8578 
- Fixed local test failure on doubleclick #8644 
- New ad loader design experiment. Please opt-in through experiment tag name `ad-loader-v1` or `ad-loader-v2`
- Refactored code to use `#insertAnalyticsElement` function. #8628 
- Work on `waitFor` support for visibility tracking.

**Comment by camelburrito** [comment]
AMP Start
 - New Preview Modal
 - Reviewed and approved Pending PRs 
 - Production Release

AMP HTML
 - Releases & Cherrypicks

**Comment by spacedino** [comment]
- ads loader refresh experiment with @zhouyx 
- ABE visual update with @sebastianbenz 
- research plan ampstart
- config plan
- working with @camelburrito to push sprint changes to prod
